ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__installation-nightly.yaml

```diff
@@ -0,0 +1,76 @@
+base_images:
+  ansible:
+    name: "4.15"
+    namespace: ocp
+    tag: ansible
+  aws-efs-csi-operator-create-efs:
+    name: "4.15"
+    namespace: ocp
+    tag: aws-efs-csi-operator-create-efs
+  cli:
+    name: "4.15"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  tools:
+    name: "4.15"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.15"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  arm64-latest:
+    candidate:
+      architecture: arm64
+      product: ocp
+      stream: nightly
+      version: "4.15"
+  latest:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.15"
+  multi-latest:
+    candidate:
+      architecture: multi
+      product: ocp
+      stream: nightly
+      version: "4.15"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: aws-ipi-preserve-ign-arm-f14
+  cron: 53 19 10,25 * *
+  steps:
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      OCP_ARCH: arm64
+    test:
+    - ref: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-aws-ipi-preserve-bootstrap-ign
+zz_generated_metadata:
+  branch: release-4.15
+  org: openshift
+  repo: openshift-tests-private
```

Comment:
I thought that we could have a separate repo, is it possible? `openshift-openshift-tests-private-release-4.15-periodics.yaml` will become huge.

Reply:
Some doubt:
1. can we freely change any repo name here?
2. once changed, does RP classifier still upload the result to RP?

Reply:
Perhaps we can reuse the verification-tests repo ?

Reply:
> 2\. once changed, does RP classifier still upload the result to RP?

@JianLi-RH can you help confirm?

Reply:
> Perhaps we can reuse the verification-tests repo ?

That is okay for me. Then we only need to confirm the 2nd question.

Reply:
Changes needed for classifier job to get this project included.